### DISCUSSION TITLE
What audio formats does the mod support?

### DISCUSSION THREAD

**Discussion by JustNikoth** [Flazhik/Jukebox#1] [Q&A] [answered]
What audio formats does the mod support?

**Comment by Flazhik** [comment, marked as answer]
mp3, wav and ogg formats are supported at this moment

**Reply by Elarrla** [reply]
hey i changed the f4 keybind to left mouse click on accident and now im screwed, i cant change it I tried uninstalling and instaling the mod but the keybing was saved somehow, do you know how to fix it or my stupidity doomed me for ever?

**Reply by Flazhik** [reply]
@Elarrla Hey! Please go to **ULTRAKILL\Preferences** folder and open **LocalPrefs.json** file in a text editor.
There you will find a string containing **cyberGrind.musicExplorer.keyBinding.CGMEMenu**. Change a value after the colon to 285 and restart Ultrakill, it'll reset hotkey back to F4.
And thank you for reporting, bad design decisions on my part, I'll fix it soon.

**Reply by Elarrla** [reply]
ok thanks!
